### DISCUSSION TITLE
文档说 _config.page.yml 是对所有自定义页面生效，自定义页面是指什么页面啊？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [ppoffice/hexo-theme-icarus#938] [Q&A]
文档关于 _config.page.yml 的介绍比较少，因此我不明白下图中自定义页面是指的什么页面？
又怎么配置为自定义页面？

![image](https://user-images.githubusercontent.com/27394631/134315450-6f9604f3-a278-4ed7-ad15-ed7a1e7031f1.png)
图片来自 [Icarus用户指南 - 主题配置 - Icarus](https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%E4%B8%8E%E4%BC%98%E5%85%88%E7%BA%A7)

**Comment by ppoffice** [comment]
https://hexo.io/zh-cn/docs/writing.html

**Reply by GanZhiXiong** [reply]
Thanks!
